deisctl/cmd/platform.go

```diff
@@ -11,12 +11,18 @@ import (
 
 // InstallPlatform loads all components' definitions from local unit files.
 // After InstallPlatform, all components will be available for StartPlatform.
-func InstallPlatform(b backend.Backend, checkKeys func() error) error {
+func InstallPlatform(b backend.Backend, checkKeys func() error, stateless bool) error {
 
 	if err := checkKeys(); err != nil {
 		return err
 	}
 
+	if stateless {
+		fmt.Println("Warning: With a stateless control plane, `deis logs` will be unavailable.")
+		fmt.Println("Additionally, components will need to be configured to use external persistent stores.")
+		fmt.Println("See the official Deis documentation for details on running a stateless control plane.")
```

Comment:
With other versions of `deisctl`, we modify `refresh-units` to pull from a specific tag if it's an official release. Perhaps we could do the same here? (i.e. v1.8.0 tagged release of deis docs for deisctl v1.8.0, otherwise latest)

This is a minor thing since stateless-platform is a very specific edge case, but just some food for thought :)